tools/topology/topology2/cavs/cavs-nocodec.conf

```diff
@@ -41,6 +41,12 @@ Define {
 	DMIC0_NAME			'NoCodec-6'
 	DMIC0_PCM_CAPS			'Passthrough Capture 13'
 	DMIC0_PIPELINE_STREAM_NAME	'copier.DMIC.14.1'
+	PLATFORM 			"none"
+}
+
+# override defaults with platform-specific config
+IncludeByKey.PLATFORM {
+	"mtl"	"platform/intel/mtl.conf"
```

Comment:
unrelated but can we enable a mixer for nocodec - just like we did for topology1? I think the topologies should look near identical.

Reply:
will do in a follow up PR